generator.py

```diff
@@ -0,0 +1,39 @@
+from random import choice
+
+from faker import Faker
+
+
+def create_random_data_list():
+    """Create  a list of 200000 dicts with random data."""
+    fake = Faker()
+    keys = ['first_name', 'last_name', 'address', 'email', 'sex', 'state',
+            'city', 'country', 'phone_number']
+    data_list = [{i: choose_data(i, fake) for i in keys} for i in
+                 range(200000)]
+    return data_list
+
+
+def choose_data(data_line, fake):
+    """Generate random data."""
+    if data_line == 'sex':
+        data = choice(['man', 'woman'])
+    else:
+        data = eval('fake.' + data_line + '()')
```

Comment:
Don't use eval. It's evil. Never use it.
Also, you can use getattr in this case.

Reply:
Thanks.